package.json

```
@@ -60,7 +60,7 @@
     "@openpgp/pako": "^1.0.12",
     "@openpgp/seek-bzip": "^1.0.5-git",
     "@openpgp/tweetnacl": "^1.0.3",
-    "@openpgp/web-stream-tools": "^0.0.11",
+    "@openpgp/web-stream-tools": "^0.0.11-patch-0",
```

In this particular case it's probably better to specify the exact version as semver considers 0.0.11 > 0.0.11-patch-0:
```suggestion
    "@openpgp/web-stream-tools": "0.0.11-patch-0",
```